lib/trino-parquet/src/test/java/io/trino/parquet/TestTupleDomainParquetPredicate.java

```diff
@@ -605,6 +606,39 @@ public void testVarcharMatchesWithDictionaryDescriptor()
         assertTrue(parquetPredicate.matches(new DictionaryDescriptor(column, Optional.of(page))));
     }
 
+    @Test
+    public void testEmptyDictionary()
+    {
+        ColumnDescriptor columnDescriptor = new ColumnDescriptor(new String[] {"path"}, BINARY, 0, 0);
+        RichColumnDescriptor column = new RichColumnDescriptor(columnDescriptor, new PrimitiveType(OPTIONAL, BINARY, "Test column"));
+        ColumnDescriptor descriptor = new ColumnDescriptor(column.getPath(), column.getPrimitiveType().getPrimitiveTypeName(), 0, 0);
```

Comment:
column.getPrimitiveType().getPrimitiveTypeName() -> column.getPrimitiveType()
Avoids usage of deprecated API